azure-common/azure/profiles/__init__.py

```diff
@@ -108,6 +108,45 @@ class KnownProfiles(Enum):
         },
         "2017-03-09-profile"
     )
+    v2018_03_01_hybrid = ProfileDefinition(
+        {
+            "azure.keyvault.KeyVaultClient":{
```

Comment:
should we include storage data plane here, though we don't have a clean story data plane at the ARM level, we are keeping the support at the PS/CLI tools . Go/Ruby/C#  SDKs right now does not include data plane (I need to check java)